.circleci/config.yml

```diff
@@ -15,6 +15,8 @@ mongo_image: &mongo_image mongo:3.6
 httpbin_image: &httpbin_image kennethreitz/httpbin@sha256:2c7abc4803080c22928265744410173b6fea3b898872c01c5fd0f0f9df4a59fb
 vertica_image: &vertica_image sumitchawla/vertica:latest
 rabbitmq_image: &rabbitmq_image rabbitmq:3.7-alpine
+zookeeper_image: &zookeeper_image confluentinc/cp-zookeeper:7.3.1
+kafka_image: &kafka_image confluentinc/cp-kafka:7.3.2
```

Comment:
Are these images used in circleci? If not, we should remove them

Reply:
Will remove them, thanks for catching that!